test/infrastructure/docker/PROJECT

```diff
@@ -0,0 +1,10 @@
+version: "2"
+domain: cluster.x-k8s.io
+repo: sigs.k8s.io/cluster-api/test/infrastructure/docker
```

Comment:
Do we know if the kubebuilder tooling currently work with this configuration? If not, we should likely file a tracking ticket

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/1526